react-commenting.js

```diff
@@ -90,16 +97,19 @@ class Board extends Component{
 
           <div className="statusDiv">
             {/* 9) Here: */}
+            //Current player is part of this destructuring state.
             The Current Player is: { currentPlayer }
```

Comment:
We did destructure current player out of state on line 79, but this line is displaying the current player for the user to see.